### PR TITLE
Don't Set Missing Thumbnail URL

### PR DESCRIPTION
**Pull request by hussam-i-am**
If no thumbnail for stream is available, don’t set it to an empty string

JW7-4184

### CODE REVIEW

**Comment by hghazzi** [comment]
test this please

**Comment by hghazzi** [comment]
test this please

**Comment by hghazzi** [comment]
test this please

**Comment by hussam-i-am** [comment]
test this please

File: src/js/uicontroller.js [hunk 1]
```diff
@@ -32,12 +32,13 @@ function applyConfig(config) {
...
         document.body.className = `theme-${UIUtil.escapeHtml(config.theme)}`;
     }
-    if (typeof (config.logoUrl) === 'string' && config.logoUrl.length > 0) {
-        Array.from(document.getElementsByClassName('logo'))
-            .forEach(element => {
-                element.style.backgroundImage = `url(${config.logoUrl})`;
-            });
-    }
+    Array.from(document.getElementsByClassName('logo')).forEach(element => {
+        if (typeof (config.logoUrl) === 'string' && config.logoUrl.length > 0) {
```
Comment (egreaves) [inline src/js/uicontroller.js]: Why not check this once outside of the loop?

Comment (hussam-i-am) [reply src/js/uicontroller.js]: can do that

**Review by egreaves — APPROVED** [review]

**Pull request merged**